orchestrator/api/api_v1/endpoints/settings.py

```diff
@@ -38,9 +38,9 @@
 async def clear_cache(name: str, background_tasks: BackgroundTasks) -> None:
     cache = Cache(Cache.REDIS, endpoint=app_settings.CACHE_HOST, port=app_settings.CACHE_PORT)
     if name == "all":
-        background_tasks.add_task(cache.delete, "/*")
+        await cache.clear(namespace="orchestrator")
```

Comment:
Are you sure of the namespace? All should flush all caches I think,

Reply:
Yes, the old implementation did not flush anything

Reply:
*this is the redis-client:
127.0.0.1:6379> keys *
1) "orchestrator:crm:surf.api.api_v1.endpoints.crmlocation_codes()[]"
2) "orchestrator:crm:surf.api.api_v1.endpoints.crmorganisations()[]"
127.0.0.1:6379> keys *
(empty array)

Reply:
the keys are empty after the flush (triggered in the client)